onesignalclient/notification.py

```diff
@@ -38,6 +37,11 @@ def include_player_ids(self):
 
     @include_player_ids.setter
     def include_player_ids(self, value):
+        """
+        Usage should be:
+            notification.include_player_ids = ['player_id_1', 'player_id_2', ...]
```

Comment:
Can we make it shorter or break the lines? :)

Ref:
`'E501:line too long (81 > 79 characters)'`